js/widgets/temperament.js

```diff
@@ -2005,7 +2005,10 @@ class TemperamentWidget {
         let p = 0;
         this.playbackForward = true;
         this._playing = !this._playing;
-
+        if (!this._playing) {
+            logo.synth.setMasterVolume(0);
```

Comment:
I am not sure we want to mess with the Master volume. It may have broader impacts. Why not just exit the loop? (See the mode widget, that does something similar.)

Reply:
Yeah, just returning out of the loop would work perfectly.
Thanks.

Reply:
I made this commit. Please review again.
Thanks.